docs/tutorial/force_calibration.rst

```diff
@@ -1,380 +1,375 @@
 
-Using `pylake`, the procedure to use active calibration is not very different from passive calibration.
-However, it does require some additional data channels as inputs.
-Instead of using the :class:`~.PassiveCalibrationModel` presented in the previous section, we now use the :class:`~lumicks.pylake.ActiveCalibrationModel`.
+    shared_parameters = {
+        "force_voltage_data": volts.data,
+        "bead_diameter": bead_diameter,
+        "temperature": 25,
+        "sample_rate": volts.sample_rate,
+        "driving_data": driving_data.data,
+        "driving_frequency_guess": 37,
+        "hydrodynamically_correct": False,
+    }
 
-In this tutorial, we're going to assume the nanostage was used as driving input::
+Next, unpack this dictionary using the unpacking operator `**`::
 
-    driving_data = f["Nanostage Position"]["Nanostage X"][start:stop]
+    >>> fit = lk.calibrate_force(**shared_parameters, active_calibration=True, distance_to_surface=distance_to_surface)
+    >>> print(fit["kappa"].value)
+    0.11662183772410809
 
-We also need to provide the sample rate at which the data was acquired, and a rough guess for the driving frequency.
-`pylake` will find an accurate estimate of the driving frequency based on this initial estimate (provided that it is close enough)::
+And compare this to the passive calibration result::
 
-    active_model = lk.ActiveCalibrationModel(driving_data.data, volts.data, driving_data.sample_rate, bead_diameter, driving_frequency_guess=37)
+    >>> fit = lk.calibrate_force(**shared_parameters, active_calibration=False, distance_to_surface=distance_to_surface)
+    >>> print(fit["kappa"].value)
+    0.11763849764570819
 
-To check the determined frequency, we can look at the determined driving frequency::
+These values are quite close.
+However, if we do not provide the height above the surface, we can see that the passive calibration result suffers much more than the active calibration result (as passive calibration fully relies on a drag coefficient calculated from the physical input parameters)::
 
-    >>> active_model.driving_frequency
-    36.95
+    >>> print(lk.calibrate_force(**shared_parameters, active_calibration=False)["kappa"].value)
+    >>> print(lk.calibrate_force(**shared_parameters, active_calibration=True)["kappa"].value)
+    0.08616565751377737
+    0.11662183772410809
 
-We can now use this model to fit the power spectrum::
+.. note::
 
-    calibration = lk.fit_power_spectrum(power_spectrum, active_model)
-    calibration
+    When hydrodynamic correction is enabled, the height parameter impacts both passive and active calibration.
+    The reason for this is that when using the more accurate theory, the expected shape of the power spectrum depends on the distance to the surface.
+    For more information on this see the :doc:`theory section on force calibration</theory/force_calibration/force_calibration>` section.
```

Comment:
this seems a bit strange. In the first sentence you talk about hydrodynamic correction, but in the 2nd distance to surface; the connection isn't really clear

Reply:
Yes. I agree. I think inverting the argument may be better here. How about:
```
    When fitting with the hydrodynamically correct model, the `distance_to_surface` parameter impacts the expected shape of the power spectrum.
    Consequently, when this model is selected, this parameter affects both passive and active calibration.
    For more information on this see the :doc:`theory section on force calibration</theory/force_calibration/force_calibration>` section.
```